package.json

```diff
@@ -41,10 +39,13 @@
   },
   "homepage": "https://github.com/asciidoctor/asciidoctor-reveal.js",
   "dependencies": {
-    "asciidoctor.js": "1.5.9",
-    "reveal.js": "3.7.0"
+    "reveal.js": "3.6.0"
```

Comment:
Why the version downgrade?

Reply:
Ooops I don't think it's intended... I will revert this change.